package.json

```diff
@@ -28,8 +28,8 @@
     "grunt-eslint": "^18.0.0",
     "grunt-karma": "2.x",
     "grunt-npm": "0.0.2",
-    "jasmine-core": "~2.4.1",
     "karma": "2.x || ",
+    "jasmine-core": "~3.0.0",
```

Comment:
Your description says "Add support for 3.0" so shouldn't this allow 2.4.1 through ~3.0.0?

Reply:
Well the spyOn() calls require 3.0 for dev, so this is ok. I guess the dep between karma-jasmine adapter and jasmine runtime is not specified in the package.json, we just cross our fingers.

Reply:
In order to test the `failFast` option, I needed to have Jasmine 3.x in the tests because of how they load and check for options. It looks like karma-jasmine has always kinda crossed its fingers about the version of jasmine-core being used.